src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -367,6 +453,20 @@ class CreatePipelineRun extends React.Component {
               />
             </FormGroup>
           )}
+          <FormGroup legendText="">
+            <KeyValueList
+              legendText="Labels"
```

Comment:
strings should be externalised using react-intl (`intl.formatMessage(...)` or the `<FormattedMessage ... />` component)